manifests/local/orchestration-modules.yaml

```diff
@@ -23,3 +26,5 @@ parameters:
     value: dags
   - name: airflow-version
     value: "2.5.1"
```

Comment:
The airflow version should "2.2.2"...not sure how this version got into the repo...